stable/nginx-ingress/templates/controller-psp.yaml

```diff
@@ -2,7 +2,7 @@
 apiVersion: extensions/v1beta1
 kind: PodSecurityPolicy
 metadata:
-  name: {{ template "nginx-ingress.fullname" . }} 
+  name: {{ template "nginx-ingress.fullname" . }}-controller
```

Comment:
What is the reasoning behind renaming the policy? I am slightly worried that renaming this might cause problems when upgrading.

Reply:
tbh, we really don't need to, it was more to make it consistent where psp-controller was used vs. psp-backend.   I can remove this rename and keep it as-is, and only append -backend to the new PSP